x/twap/api_test.go

```diff
@@ -103,13 +120,30 @@ func makeSimpleTwapInput(startTime time.Time, endTime time.Time, isQuoteTokenA b
 	return getTwapInput{1, quoteAssetDenom, baseAssetDenom, startTime, endTime}
 }
 
+func makeSimpleTapTwapInput(startTime time.Time, endTime time.Time, baseQuoteAB, baseQuoteCA, baseQuoteBC bool) []getTwapInput {
```

Comment:
What does "Tap" mean? I'm guessing something to do with 3-assets?

Please consider being more specific with the abbreviation

Reply:
"ThreeAssetPool", do you suggest I just error on the side of verbosity and change every reference of "tap" to "threeAssetPool"? The more I think about it the more I feel I should change it to reduce cognitive overhead

Reply:
I decided to change `tap` to threeAsset, hopefully that is a good middle ground!